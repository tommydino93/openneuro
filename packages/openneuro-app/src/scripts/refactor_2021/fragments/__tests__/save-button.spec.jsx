import React from 'react'
import { shallow } from 'enzyme'
import SaveButton from '../edit-button.jsx'

describe('SaveButton component', () => {
  it('renders with default props', () => {
    const wrapper = shallow(<SaveButton />)
    expect(wrapper).toMatchSnapshot()
  })
})
