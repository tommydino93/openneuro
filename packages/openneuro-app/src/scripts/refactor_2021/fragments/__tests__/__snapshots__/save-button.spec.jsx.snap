// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SaveButton component renders with default props 1`] = `
<WarnButton
  action={[Function]}
  cancel={
    <i
      className="fa fa-times"
    />
  }
  confirm={
    <i
      className="fa fa-check"
    />
  }
  icon="fa-edit"
  message="Edit"
  tooltip={null}
  warn={false}
/>
`;
